brainstormforce/astra
# AST-2228 - fix: page title misalignment with page container on full width stretched

### Description

- Fix: Page title is expected to align with page container.
### Screenshots

https://d.pr/i/0vUk54

### Types of changes


- Fix: Page title is expected to align with page container.



### How has this been tested?

- Create a new page using Spectra, Gutenberg.
- Make sure page title is visible.
- Add some content to page.
- Select Full Width Stretched Layout from Astra Metabox settings > Content Layout.
- Click Publish, Page title should be aligned with Page container.

### Checklist:
